ska_sync/ska_sync_config

```diff
@@ -50,6 +50,31 @@ file_sync:
   acq_stats:
     - acq_stats.h5
 
+  # Minimal set of eng-archive files for doing ACA and ACIS load review
+  eng_archive:
+    - data/orbitephem0/colnames.pickle
```

Comment:
I think these are in the "order that the tool complained about them missing".  I don't know if they should be alpha or some such.  Doesn't matter to me.